engine/runner/src/main/java/org/enso/runner/Main.java

```diff
@@ -1236,14 +1276,18 @@ private void launch(String[] args) throws IOException, InterruptedException, URI
 
     var logMasking = new boolean[1];
     var logLevel = setupLogging(line, logMasking);
+    var props = parseSystemProperties(line);
 
     if (line.hasOption(JVM_OPTION)) {
       var jvm = line.getOptionValue(JVM_OPTION);
       var current = System.getProperty("java.home");
       if (jvm == null) {
         jvm = current;
       }
-      if (current == null || !current.equals(jvm)) {
+      var shouldLaunchJvm = current == null || !current.equals(jvm);
+      if (!shouldLaunchJvm) {
+        println(JVM_OPTION + " option has no effect - already running in JVM " + current);
```

Comment:
Just a helpful message that `--jvm` is ignored if you are already running in the JVM.